config/index.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
 
     // Various Dev Server settings
     host: 'localhost', // can be overwritten by process.env.HOST
-    port: 8080, // can be overwritten by process.env.PORT, if port is in use, a free one will be determined
+    port: 3000, // can be overwritten by process.env.PORT, if port is in use, a free one will be determined
```

Comment:
@rodrigopavezi I don't know a great deal about ports so genuinely curious why 3000. Might be a teachable moment if you wouldn't mind? :)

Reply:
@jungeunyoon it is because LMS is using 8080 for the primus websocket. So I changed the dev webpack server to 3000. It will not impact any production code. It is just for testing locally and run integration tests. I have seen other webservers to use 3000 so that is why I have chosen it. cheers

Reply:
@rodrigopavezi Ah okay knew about LMS -> 8080 but wasn't aware about of webservers using 3000. Thanks for the info! 😄

Reply:
👍